### PR TITLE
Added community providers

### PR DESCRIPTION
**Pull request by patelvivekdev**
I added two custom providers: Voyage AI Provider and Mixedbread AI Provider. 

Both providers currently support embedding models. Reranking support will be added after the reranking feature is updated.

[Mixedbread AI](https://www.npmjs.com/package/mixedbread-ai-provider)

[Voyage AI](https://www.npmjs.com/package/voyage-ai-provider)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-packages/ai/README.md
\ No newline at end of file
+packages/ai/README.md
```
Comment (lgrammel) [inline README.md:1]: revert

File: content/providers/03-community-providers/06-mixedbread-ai.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---
+title: Mixedbread AI Provider
+description: Learn how to use the Mixedbread AI provider.
+---
+
+# Mixedbread AI Provider
+
+[patelvivekdev/mixedbread-ai-provider](https://github.com/patelvivekdev/mixedbread-ai-provider) is a community provider that uses [Mixedbread](https://www.mixedbread.ai/) to provide Embedding support for the AI SDK.
```
Comment (lgrammel) [inline content/providers/03-community-providers/06-mixedbread-ai.mdx:8]: how are the 2 providers differnt? why 2 providers?

File: content/providers/03-community-providers/06-mixedbread-ai.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## Setup
+
+The Ollama provider is available in the `mixedbread-ai-provider` module. You can install it with
```
Comment (lgrammel) [inline content/providers/03-community-providers/06-mixedbread-ai.mdx:12]: Ollama?

File: content/providers/03-community-providers/07.voyage-ai.mdx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+```ts
+import { createVoyage } from 'voyage-ai-provider';
+
+const voyage = createVoyage({
+  baseURL: 'https://api.voyageai.com/v1',
+  apiKey: process.env.VOYAGE_API_KEY,
+});
+```
+
+You can use the following optional settings to customize the Ollama provider instance:
```
Comment (lgrammel) [inline content/providers/03-community-providers/07.voyage-ai.mdx:45]: why does it say ollama everywhere

Comment (patelvivekdev) [reply content/providers/03-community-providers/07.voyage-ai.mdx]: > why does it say ollama everywhere
Sorry, I used Ollama as a reference to write that and forgot to update it.

File: README.md (hunk 1 above)
Comment (patelvivekdev) [reply README.md:1]: I haven't changed anything in that file.

**Comment by lgrammel** [comment]
https://github.com/vercel/ai/pull/3862

**Pull request closed without merging**